### ISSUE TITLE
[Deque Analysis] Community/Collection Homepage "serious" issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1178]
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority. 

## Deque Analysis Summary

Community & Collection homepages have a total of 8 serious issues (not including color contrast which are handled in other tickets).  These have been grouped into one ticket because they are similar issues on both pages.

1. (ID 469493, 469644) (BOTH pages) ~~"Text should not be marked as a heading" on the "Permanent URI for this.." text on both pages.  In both cases this text is an `<h2>`, but it is obviously not heading text.  It probably should just be normal text (e.g. `<p>` tag or similar)~~
    * FIXED BY #2683
3. (ID 469552, 469655) (BOTH pages) "Selected state of the element is missing or incorrect."  on the currently selected browse by option (e.g. "Subcommunities and Collections" on Community homepage, "Recent Submission" on Collection homepage).  In both cases, the selected option is displayed visually, but not noted in ARIA tags. 
    * Fix: Use the `aria-current` attribute to indicate the element that represents the current item within a container or set of related elements, such as the currently selected page, step, location, date, or time. https://www.w3.org/TR/wai-aria-1.1/#aria-current
    * **MOVED TO #2700**
4. (ID 470094, 470956) ~~(BOTH pages) "Link contrast is not at least 3:1 with surrounding text." on the Handle link at the top of both pages.~~
    * ~~"Link text and static text are differentiated only by color. The contrast ratio between default link text and surrounding text is not at least 3:1." Link text: `#207698`, Surrounding text : `#444444`, The contrast ratio is: 1.9:1.~~
    * ~~Fix is to either add an underline to link text (to distinguish it), or increase contrast using https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI~~
    * FIXED BY #2587
5. (ID 470257) ~~Community homepage doesn't appear to have a `<title>`.  It keeps the title of the last page.  (This issue doesn't appear on the Collection hompage)~~
6. (ID 470963) ~~"Content is lost at 320px width" occurs on the pagination bar (page numbers) when a large number of pages exist on a Collection homepage (e.g. https://demo7.dspace.org/collections/282164f5-d325-4740-8dd1-fa4d6d3e7200).  Simply put, this bar doesn't collapse nice on small screens...likely it should add a "..." between most pages as the screen gets smaller.~~
     * ~~NOTE: This is obviously a general pagination issue that can affect this same pagination bar on other pages.~~
     * **UPDATE: No longer reproducible in 7.6.1**

Full list of issues is viewable at (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=50&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=3&filter%5Btype%5D=issue&filter%5Bpage_number%5D=4&filter%5Bpage_number%5D=5

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by tdonohue** [comment]
Updated this ticket to be closed by #2587, as that will solve the outstanding Color Contrast issues.

Once Color Contrast is solved, the only remaining issue involves the Browse By Tabs.  I've moved that issue to a separate ticket so that it can be addressed/tracked separate from this larger ticket.  See #2700 

In summary this ticket will be closed as fixes as soon as #2587 is merged.  #2700 will be addressed in a separate ticket.
